chemie/events/templates/events/admin_list.html

```diff
@@ -197,35 +203,63 @@
         <div class="determinate" style="width:{{ percentage_paid }}%"></div>
       </div>
     </p>
-    <div align="right" id="paidcounter">{{ total_paid }} har betalt • {{ total_not_paid }} har ikke betalt</div>
+    <div align="right" id="paidcounter"></div>
+    <script>
+      var totalPaid = 0;
+      var totalNotPaid = 0
+      for (var key in paymentList) {
+    // check if the property/key is defined in the object itself, not in parent
+        if (paymentList.hasOwnProperty(key)) {
+            if (paymentList[key][1]){
+              totalPaid += 1
+            } else {
+              totalNotPaid += 1
+            }
+        }
+    }
+
+    document.getElementById("paidcounter").innerHTML = "{{ total_paid }} har betalt • {{ total_not_paid }} har ikke betalt"
+    </script>
     {% endif %}
   </div>
 </div>
 {% endblock content %}
 
 {% block footer_script %}
 <script src="{% static 'js/boolean_status_widget.js' %}" type='text/javascript'></script>
-  <script>
-      $(document).ready(function () {
-          $('.payment').on('click', function (event) {
-              //$('.payment').live('click', function(event){
-              event.preventDefault();
-              var myID = $(this).attr("id").split('-')[1];
-              var csrftoken = getCookie('csrftoken');
+<script>
+    $(document).ready(function () {
+        $('.payment').on('click', function (event) {
+            //$('.payment').live('click', function(event){
+            event.preventDefault();
+            var myID = $(this).attr("id").split('-')[1];
+            var csrftoken = getCookie('csrftoken');
+
+            var changePayment = true
+            if (paymentList[myID][1]){
+              changePayment = confirm("Endre betalingsstatus for " +  paymentList[myID][0] + " fra betalt til ikke betalt?")
+            }
+            if (changePayment){
+              paymentList[myID][1] = !paymentList[myID][1]
               $.ajax({
-                  type: "GET",
-                  url: "/events/social/adminlist/paymentstatus/" + myID,
-                  datatype: "json",
+                type: "GET",
```

Comment:
This should actually be a POST request. Should be looked into while we are working on the file

Reply:
Jeg stusset litt på det da jeg så fila at det var en GET-request, kan mekke en POST-only method i views

Reply:
Also has to swap ```type: "GET"``` to ```type: "POST"```